_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+---
+layout:         pattern
+title:          Tables
+date:           2015-12-11 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- buttons
+
+slug:           tables
+
+url_styles:     "components/_tables.scss"
```

Comment:
Looks like you copied from a bad/old example pattern. This changed to be ``"patterns/_[patternname].scss"`` a little bit ago. Sorry for the screw up.